instance.go

```diff
@@ -9,7 +9,7 @@ import (
 )
 
 type Instance struct {
-	InstanceID, PrivateIP, State           string
+	InstanceId, PrivateIP, State           string
```

Comment:
I just spotted that I'd also changed the Instance type in this package. I think that was probably a mistake.